src/packages/@ncigdc/components/Oncogrid/OncogridWrapper.js

```diff
@@ -399,7 +400,15 @@ const OncoGridWrapper = compose(
             <div style={{ flexGrow: 1 }} className="oncogrid-legend">
               {heatMapMode
                 ? <StepLegend rightLabel="More Mutations" />
-                : <SwatchLegend colorMap={colorMap} />}
+                : <SwatchLegend
+                    colorMap={Object.entries(colorMap).reduce(
+                      (acc, [key, val]) =>
+                        Object.assign(acc, {
+                          [key.replace("_variant", "")]: val
+                        }),
+                      {}
+                    )}
+                  />}
```

Comment:
in the lodash/fp version:

`The iteratee of mapKeys is capped to one argument: (key)`